### PR TITLE
Initial migration

### PR DESCRIPTION
**Pull request by tijsverkoyen**
* This will make it easier to set up the project
* This will writing the first migration more logical

This will fix https://github.com/sumocoders/Framework/issues/40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,8 @@
 This is the SumoCoders Symfony Framework. You can install it using
 
     composer create-project -s dev sumocoders/framework .
+    app/console doctrine:migrations:migrate
```
Comment (WouterSioen) [inline README.md]: mss in de "post-create-project-cmd" hook van composer steken?

File: app/migrations/Version20150331151848.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+     */
+    public function up(Schema $schema)
+    {
+        $this->abortIf(
+            $this->connection->getDatabasePlatform()->getName() != 'mysql',
+            'Migration can only be executed safely on \'mysql\'.'
+        );
+
+        $this->addSql(
+            'CREATE TABLE User (id INT AUTO_INCREMENT NOT NULL, username VARCHAR(255) NOT NULL, username_canonical VARCHAR(255) NOT NULL, email VARCHAR(255) NOT NULL, email_canonical VARCHAR(255) NOT NULL, enabled TINYINT(1) NOT NULL, salt VARCHAR(255) NOT NULL, password VARCHAR(255) NOT NULL, last_login DATETIME DEFAULT NULL, locked TINYINT(1) NOT NULL, expired TINYINT(1) NOT NULL, expires_at DATETIME DEFAULT NULL, confirmation_token VARCHAR(255) DEFAULT NULL, password_requested_at DATETIME DEFAULT NULL, roles LONGTEXT NOT NULL COMMENT \'(DC2Type:array)\', credentials_expired TINYINT(1) NOT NULL, credentials_expire_at DATETIME DEFAULT NULL, UNIQUE INDEX UNIQ_2DA1797792FC23A8 (username_canonical), UNIQUE INDEX UNIQ_2DA17977A0D96FBF (email_canonical), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB'
```
Comment (WouterSioen) [inline app/migrations/Version20150331151848.php]: Ik zou die query wa schoner uitlijnen, maar is niet per sé nodig.

File: README.md (hunk 1 above)
Comment (tijsverkoyen) [inline README.md]: Fixed in b780de46be40e5c402218448f4d4a724a4699567

File: app/migrations/Version20150331151848.php (hunk 2 above)
Comment (tijsverkoyen) [inline app/migrations/Version20150331151848.php]: Fixed in b780de46be40e5c402218448f4d4a724a4699567

**Comment by WouterSioen** [comment]
:shipit:

**Pull request merged**